wiki/osu!stream/en.md

```diff
@@ -1,15 +1,15 @@
-<!-- TODO: some updates and needs a proper end of life message. it's also open source now -->
-
 # osu!stream
 
 ![](img/Os-Logo.jpg "osu!stream logo")
 
-**osu!stream** is a version of osu! built from scratch that is made especially for touch-enabled devices. It is currently available for Apple's [iOS](http://en.wikipedia.org/wiki/IOS) platform only, but an [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) port is planned. It introduces a brand new game-play system such as the hold circle and connected beats.
+**osu!stream** is a version of osu! built from scratch that is made especially for touch-enabled devices. It is currently available for Apple's [iOS](http://en.wikipedia.org/wiki/IOS) platform only. The development ended in 2020 and is to be seen as finished. An [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) port was planned but not implemented yet. For more information, read the [blog post](https://blog.ppy.sh/osu-stream-2020-release/) about its last release.
 
-It does not support classic osu! beatmaps. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a stricter level of standards. Both free and paid maps are available at the "Store".
+It introduces a brand new gameplay system such as the hold circle and connected beats. It does not support classic osu! beatmaps. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a stricter level of standards. All maps are available for free at the "Store".
 
 [osu!stream was elaborated in osu!monthly \#5 along with an interview.](https://osu.ppy.sh/community/forums/topics/59924)
 
+The source code is available for everyone curious enough on its [GitHub repository](https://github.com/ppy/osu-stream).
```

Comment:
```suggestion
The source code was made available in 2020 under a permissive licence on [GitHub](https://github.com/ppy/osu-stream).
```

Reply:
In my review, I have rearranged some information around the neighbourhood of this paragraph to be more cohesive

```suggestion
Development for this version ended in 2020. For more information, read the [blog post](https://blog.ppy.sh/osu-stream-2020-release/) about its last release. The source code has been made available in 2020 under a permissive license on [GitHub](https://github.com/ppy/osu-stream) for anyone curious, and all previously paid maps are available for free from the in-game "Store".
```